### PR TITLE
[Snyk] Security upgrade axios from 0.21.4 to 1.6.0

### PR DESCRIPTION
**Pull request by ericboucher**
<p>This PR was automatically created by Snyk using the credentials of a real user.</p><br /><h3>Snyk has created this PR to fix one or more vulnerable packages in the `yarn` dependencies of this project.</h3>

#### Changes included in this PR

- Changes to the following files to upgrade the vulnerable dependencies to a fixed version:
    - packages/website/package.json

#### Note for [zero-installs](https://yarnpkg.com/features/zero-installs) users

If you are using the Yarn feature [zero-installs](https://yarnpkg.com/features/zero-installs) that was introduced in Yarn V2, note that this PR does not update the `.yarn/cache/` directory meaning this code cannot be pulled and immediately developed on as one would expect for a zero-install project - you will need to run `yarn` to update the contents of the `./yarn/cache` directory. 
If you are not using zero-install you can ignore this as your flow should likely be unchanged.

<details>
<summary>⚠️ <b>Warning</b></summary>

```
Failed to update the yarn.lock, please update manually before merging.
```
</details>

##### With an upgrade:
Severity                   | Priority Score (*)                   | Issue                   | Breaking Change                   | Exploit Maturity
:-------------------------:|-------------------------|:-------------------------|:-------------------------|:-------------------------
high severity  |  **748/1000**  <br/> **Why?** Proof of Concept exploit, Recently disclosed, Has a fix available, CVSS 7.1  | Cross-site Request Forgery (CSRF) <br/>[SNYK-JS-AXIOS-6032459](https://snyk.io/vuln/SNYK-JS-AXIOS-6032459) |  Yes  | Proof of Concept 

(*) Note that the real score may have changed since the PR was raised.

Check the changes in this PR to ensure they won't cause issues with your project.

------------

**Note:** *You are seeing this because you or someone else with access to this repository has authorized Snyk to open fix PRs.*

For more information:  
🧐 [View latest project report](https://app.snyk.io/org/ericboucher/project/f27aee1e-8729-4d45-b725-70917e80354b?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr)

🛠 [Adjust project settings](https://app.snyk.io/org/ericboucher/project/f27aee1e-8729-4d45-b725-70917e80354b?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr/settings)

📚 [Read more about Snyk's upgrade and patch logic](https://support.snyk.io/hc/en-us/articles/360003891078-Snyk-patches-to-fix-vulnerabilities)

[//]: # (snyk:metadata:{"prId":"4c57e320-a0e3-4de8-a41e-7fc8b1742a6e","prPublicId":"4c57e320-a0e3-4de8-a41e-7fc8b1742a6e","dependencies":[{"name":"axios","from":"0.21.4","to":"1.6.0"}],"packageManager":"yarn","projectPublicId":"f27aee1e-8729-4d45-b725-70917e80354b","projectUrl":"https://app.snyk.io/org/ericboucher/project/f27aee1e-8729-4d45-b725-70917e80354b?utm_source=github&utm_medium=referral&page=fix-pr","type":"auto","patch":[],"vulns":["SNYK-JS-AXIOS-6032459"],"upgrade":["SNYK-JS-AXIOS-6032459"],"isBreakingChange":true,"env":"prod","prType":"fix","templateVariants":["updated-fix-title","pr-warning-shown","priorityScore"],"priorityScoreList":[748],"remediationStrategy":"vuln"})

---

**Learn how to fix vulnerabilities with free interactive lessons:**

 🦉 [Cross-site Request Forgery (CSRF)](https://learn.snyk.io/lesson/csrf-attack/?loc&#x3D;fix-pr)

### CODE REVIEW

File: packages/website/src/common/SurveyForm/__snapshots__/index.test.tsx.snap [hunk 1]
```diff
@@ -1089,7 +1089,7 @@ exports[`SurveyForm calls onSubmit with the form data on submit 1`] = `
                         role="presentation"
                       >
                         <mock-iconbutton
-                          classname="MuiPickersDay-day MuiPickersDay-current MuiPickersDay-daySelected"
+                          classname="MuiPickersDay-day"
```
Comment (ericboucher) [inline packages/website/src/common/SurveyForm/__snapshots__/index.test.tsx.snap:1092]: @echaidemenos could you investigate why this change occured / is needed?
Is it an edge case with dates?

File: packages/website/src/routes/SiteRoutes/SiteApplication/index.tsx [hunk 2]
```diff
@@ -87,7 +87,7 @@ const Apply = ({ match, classes }: ApplyProps) => {
           setMessage(null);
         })
         .catch(({ response }: AxiosError) => {
-          if (response?.data.statusCode === 403) {
+          if (response?.status === 403) {
```
Comment (ericboucher) [inline packages/website/src/routes/SiteRoutes/SiteApplication/index.tsx:90]: @echaidemenos will this work?

File: packages/website/src/common/SurveyForm/__snapshots__/index.test.tsx.snap (hunk 1 above)
Comment (echaidemenos) [reply packages/website/src/common/SurveyForm/__snapshots__/index.test.tsx.snap:1092]: I have pushed some relevant changes for this issue in https://github.com/aqualinkorg/aqualink-app/pull/938. We can ignore this here and keep the from the other PR

File: packages/website/src/common/SurveyForm/__snapshots__/index.test.tsx.snap [hunk 3]
```diff
@@ -115,7 +115,7 @@ exports[`SurveyForm calls onSubmit with the form data on submit 1`] = `
                         class="MuiPickersSlideTransition-slideExit MuiPickersSlideTransition-slideExitActiveLeft-right"
                         variant="body1"
                       >
-                        October 2023
+                        November 2023
```
Comment (ericboucher) [inline packages/website/src/common/SurveyForm/__snapshots__/index.test.tsx.snap:118]: The fact that this changes worries me. We need to mock a date somewhere to prevent these snapshots to update every day/month @echaidemenos
